apps/formbricks-com/app/docs/self-hosting/external-auth-providers/page.mdx

```diff
@@ -0,0 +1,62 @@
+export const meta = {
+  title: "Integrating Google OAuth with Formbricks",
+  description: "Enable Google OAuth for your Formbricks instance. Follow our guide for a seamless user login experience.",
+};
+
+## Google OAuth Authentication
+
+Integrating Google OAuth with your Formbricks instance allows users to log in using their Google credentials, ensuring a secure and streamlined user experience. This guide will walk you through the process of setting up Google OAuth for your Formbricks instance.
+
+### Requirements
+
+- A Google Cloud Platform (GCP) account.
+- A Formbricks instance running and accessible.
+
+### Steps
+
+1. **Create a GCP Project**:
+   - Navigate to the [GCP Console](https://console.cloud.google.com/).
+   - From the projects list, select a project or create a new one.
+
+2. **Setting up OAuth 2.0**:
+   - If the **APIs & services** page isn't already open, open the console left side menu and select **APIs & services**.
+   - On the left, click **Credentials**.
+   - Click **New Credentials**, then select **OAuth client ID**.
```

Comment:
Here instead of `New Credentials` we can be more specific and write `Create Credentials` 😊

<img width="644" alt="Screenshot 2023-10-25 at 10 53 59 AM" src="https://github.com/formbricks/formbricks/assets/67850763/8ab49c34-f442-4962-8614-f12d25698812">